src/main/resources/views/CommercialVehicles/VT12.hbs

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```

Comment:
I understand the logic, as you want to capture both VT(G)12 and VT(P)12, however to avoid confusion I think it may be simpler refactoring this to abandoned, in the same way there re `rwt`, `plates` and `adr` files in there.

There may already be a document called VT12, that would be a reason for abandon for a non-commercial vehicle, as their conventions on the MOT side are VT20, VT30, etc